client.go

```diff
@@ -46,11 +47,15 @@ func (c *Client) newRequest(method, path string, body io.Reader) (*http.Request,
 	if c.key != "" {
 		req.Header.Add("Authorization", c.key)
 	}
-	if body == nil {
-		fmt.Println("request to ", url.String(), "with no body data")
-	} else {
-		fmt.Println("request to ", url.String(), "with body data", body.(*bytes.Buffer).String())
+
+	if os.Getenv("GF_LOG") != "" {
```

Comment:
Oh, yuck... debug code got merged here, I guess.

This should probably either be removed or switch to `log.Println` instead. The latter would be consistent with how things are done in other clients Terraform uses, and this library is (to be honest) primarily for Terraform's benefit.

Reply:
@apparentlymart That seems reasonable. I pushed another commit.